cmd/kpod/pull.go

```diff
@@ -49,6 +49,7 @@ func pullCmd(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrapf(err, "could not create runtime")
 	}
+	defer runtime.Shutdown()
```

Comment:
Is this needed in any other kpod * command?

Reply:
Nalin hit a bunch of them in a previous PR, but it's probably worth doing an audit to verify we shut everything down properly

Reply:
This is the only one using runtime, all other code that is being moved to runtime by @umohnani8 should have this code.

Reply:
I will keep a look out for this